README.md

```diff
@@ -20,14 +20,38 @@ Your goal is to produce shapes by cutting, rotating, merging and painting parts
 5. If not, file a new issue here: https://github.com/tobspr/shapez.io/issues/new
 6. I will then have a look (This can take days or weeks) and convert it to trello, and comment with the link. You can then vote there ;)
 
-## Building
-
--   Make sure `ffmpeg` is on your path
--   Install Node.js and Yarn
--   Install Java (required for textures)
--   Run `yarn` in the root folder
--   Cd into `gulp` folder
--   Run `yarn` and then `yarn gulp` - it should now open in your browser
+## Building (for windows)
+
+(credit to Hyperion-21)
+
+**SETUP**
+1. Download FFmpeg at https://www.gyan.dev/ffmpeg/builds/ffmpeg-release-full.7z
+2. Extract the .zip onto desktop, rename it to FFmpeg
+3. Move the folder into `C:\Program Files (x86)`, anywhere works but there is convienient
```

Comment:
`C:\Program Files` may be better because FFmpeg is 64-bit (to avoid confusion)
(Appears throughout file)